providers/prometheus/go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/grpc-ecosystem/go-grpc-middleware/providers/prometheus
+
+go 1.15
```

Comment:
I think we agreed we put all of this into `openmetrics` directory no? Why it is in Prometheus now?

Reply:
Thanks, I was confused with the naming, changing to openmetrics!

Reply:
Done